src/codegen/opt/build_opencl_off.cc

```diff
@@ -17,5 +17,14 @@ Module OpenCLModuleCreate(
   return codegen::DeviceSourceModuleCreate(data, fmt, fmap, "opencl");
 }
 
+Module SDAccelModuleCreate(
```

Comment:
make SDAccel into a subfolder of runtime/opencl, add an optional configuration flag in cmake(USE_SDACCEL). This is mainly to minimize the runtime binary size